Lib/copy.py

```diff
@@ -56,6 +56,11 @@ class Error(Exception):
     pass
 error = Error   # backward compatibility
 
+class _NoValue:
+    pass
+
+NoValue = _NoValue()
```

Comment:
```suggestion
NoValue = object()
```
Nit (feel free to ignore): I think, there's no need in extra class for sentinel object.